zip-0032.rst

```diff
@@ -361,6 +428,43 @@ Orchard child key derivation
 - Use :math:`I_L` as the child spending key :math:`\mathsf{sk}_{i}`.
 - Use :math:`I_R` as the child chain code :math:`\mathsf{c}_i`.
 
+Orchard internal key derivation
+-------------------------------
+
+As in the case of Sapling, for each external address, we want to produce another address
```

Comment:
Same case here; I don't think we have any reason to have more than one internal address per spending key.